code/modules/cargo/packs/livestock.dm

```diff
@@ -31,18 +31,21 @@
 	name = "Cat Crate"
 	desc = "The cat goes meow! Comes with a collar and a nice cat toy! Cheeseburger not included."//i can't believe im making this reference
 	cost = CARGO_CRATE_VALUE * 10 //Cats are worth as much as corgis.
-	contains = list(/mob/living/simple_animal/pet/cat,
+	contains = list(/mob/living/basic/pet/cat,
```

Comment:
can you change this to be properly multilined because i don't think anyone else will ever do it